src/test/java/uk/gov/hmcts/reform/ccd/data/CaseDataRepositoryTest.java

```diff
@@ -0,0 +1,85 @@
+package uk.gov.hmcts.reform.ccd.data;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.reform.ccd.config.TestConfig;
+import uk.gov.hmcts.reform.ccd.data.entity.CaseDataEntity;
+
+import java.util.List;
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.ccd.fixture.TestData.DELETABLE_CASE_TYPE;
+import static uk.gov.hmcts.reform.ccd.fixture.TestData.JURISDICTION;
+import static uk.gov.hmcts.reform.ccd.fixture.TestData.YESTERDAY;
+
+@DataJpaTest
+@ExtendWith(SpringExtension.class)
+@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
+@EnableJpaRepositories(basePackages = {"uk.gov.hmcts.reform.ccd.data"})
+@EntityScan(basePackages = {"uk.gov.hmcts.reform.ccd.data.entity"})
+@ActiveProfiles("unittest")
+@TestPropertySource(properties = {
+    "spring.jpa.hibernate.ddl-auto=create-drop",
+    "spring.liquibase.enabled=false",
+    "spring.flyway.enabled=true"
+})
+@ContextConfiguration(classes = {TestConfig.class})
+public class CaseDataRepositoryTest {
+
+    @Autowired
+    private CaseDataRepository caseDataRepository;
+
+    @BeforeEach
+    public void setUp() {
+        caseDataRepository.deleteAll();
+        CaseDataEntity caseDataEntity = new CaseDataEntity();
+        caseDataRepository.save(getCaseDataEntity(1L));
+        caseDataRepository.save(getCaseDataEntity(2L));
+    }
+
+    @Test
+    void testFindExpiredCases() {
```

Comment:
You are searcing for expired cases but there is no other unexpired cases.

Reply:
Other unexpired cases has been added